pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/SimpleActivityRevisionTask.java

```diff
@@ -0,0 +1,335 @@
+import org.broadinstitute.ddp.model.user.User;
+import org.broadinstitute.ddp.studybuilder.ActivityBuilder;
+import org.broadinstitute.ddp.util.ConfigUtil;
+import org.broadinstitute.ddp.util.GsonUtil;
+import org.jdbi.v3.core.Handle;
+import org.jdbi.v3.sqlobject.SqlObject;
+import org.jdbi.v3.sqlobject.config.RegisterConstructorMapper;
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.statement.SqlQuery;
+
+@Slf4j
+public class SimpleActivityRevisionTask implements CustomTask {
+    private static final String VARIABLES_UPD = "variables-update";
+
+    private static final String STUDY_UPDATES = "activity-updates";
+    private static final String BLOCK_KEY = "blockNew";
+    private static final String BLOCK_UPDATES = "block-updates";
+    private static final String OLD_TEMPLATE_KEY = "old_template_search_text";
+    private static final String VARIABLES_UPD_QS = "question-variables-update";
+
+    private static final Gson gson = GsonUtil.standardGson();
+
+    private String dataFile;
+
+    private Config dataCfg;
+    private Config varsCfg;
+    private Path cfgPath;
+    private Instant timestamp;
+    private Config cfg;
+    private Config studyCfg;
+
+    private ActivityDao activityDao;
+    private TemplateDao templateDao;
+    private SimpleActivityRevisionTask.SqlHelper sqlHelper;
+    private JdbiVariableSubstitution jdbiVarSubst;
+    private JdbiRevision jdbiRevision;
+    private JdbiBlockContent jdbiBlockContent;
+
+    private StudyDto studyDto;
+
+    private User adminUser;
+
+    @Override
+    public void consumeArguments(String[] args) throws ParseException {
+        log.info(String.join(", ", args));
+        CommandLineParser parser = new DefaultParser();
+        CommandLine cmd = parser.parse(new Options(), args);
+        String[] positional = cmd.getArgs();
+        if (positional.length < 1) {
+            throw new ParseException("Patch File is required to run.");
+        }
+        log.info(positional[0]);
+        this.dataFile = positional[0];
+
+        File file = cfgPath.getParent().resolve(this.dataFile).toFile();
+        if (!file.exists()) {
+            throw new DDPException("Data file is missing: " + file);
+        }
+        dataCfg = ConfigFactory.parseFile(file).resolveWith(varsCfg);
+
+        cfg = studyCfg;
+        timestamp = Instant.now();
+    }
+
+    @Override
+    public void init(Path cfgPath, Config studyCfg, Config varsCfg) {
+        this.cfgPath = cfgPath;
+        this.varsCfg = varsCfg;
+        this.studyCfg = studyCfg;
+    }
+
+    @Override
+    public void run(Handle handle) {
+        this.adminUser = handle.attach(UserDao.class).findUserByGuid(cfg.getString("adminUser.guid")).get();
+
+        this.studyDto = handle.attach(JdbiUmbrellaStudy.class).findByStudyGuid(dataCfg.getString("study.guid"));
+
+        List<? extends Config> activityUpdateConfigs = dataCfg.getConfigList(STUDY_UPDATES);
```

Comment:
`dataCfg.getConfigList` does not handle exception. And `List<? extends Config> configList` could be initialized here.